Kwenta/kwenta
# Clarify User Interaction for sUSD Trading

## Description
- [X] Change the market name format from e.g. ETH-Perp to ETH/sUSD (in market dropdown and positions table)
- [x] Add a CTA to available margin are when the user has no sUSD available
- [x] The CTA should open the bridge modal
- [x] Add the bridge step back to the cross margin onboarding flow when the user has no sUSD

## Related issue
closes #2303 

## Motivation and Context


## How Has This Been Tested?
Case 1. User without sUSD in wallet
- See the CTA and click on the `Get sUSD` button to open the Deposit modal with socket integration

Case 2. User with sUSD in wallet or idle margin in smart margin account
- See the manage balance as usual

## Screenshots (if appropriate):
![image](https://github.com/Kwenta/kwenta/assets/4819006/76247d68-9b0f-42c1-91d2-347e3d0fb65d)
![image](https://github.com/Kwenta/kwenta/assets/4819006/6fee86e9-2354-4e3b-b477-6dd0634a9717)
![image](https://github.com/Kwenta/kwenta/assets/4819006/6ef6d2dc-78f0-4238-ae93-38734c6274ba)
